model/config.go

```diff
@@ -838,7 +848,10 @@ func (s *SqlSettings) SetDefaults() {
 	}
 
 	if s.AtRestEncryptKey == nil || len(*s.AtRestEncryptKey) == 0 {
-		s.AtRestEncryptKey = NewString(NewRandomString(32))
+		s.AtRestEncryptKey = NewString("")
+		if !isNew {
```

Comment:
Let's put the assignment above inside `isNew` and then this inside an `else` block, so as to avoid allocating a string only to immediately replace it with another.